### ISSUE TITLE
How to verify bot session

### ISSUE THREAD

**Issue by zatevakhin** [nexy7574/nio-bot#26]
I have used provided example to create a simple bot. But can't get bot verified. I have tried to put this stuff together at 4am, so maybe missed something. 

```python
import niobot
import logging

logging.basicConfig(level=logging.INFO)

client = niobot.NioBot(
    homeserver="https://matrix.org",
    user_id="@<me>:matrix.org",
    device_id="container-svc",
    store_path='./store',
    command_prefix="!",
    case_insensitive=True,
)

@client.on_event("ready")
async def on_ready(sync_result: niobot.SyncResponse):
    print("Logged in!")

@client.on_event("message")
async def on_message(room: niobot.MatrixRoom, message: niobot.RoomMessage):
    print(message)

@client.command()
async def ping(ctx: niobot.Context):
    latency = ctx.latency
    await ctx.respond(f"Pong! {latency:.2f}ms")

@client.command()
async def echo(ctx: niobot.Context, *, message: str):
    await ctx.respond("You said: " + message)

client.run(access_token="<my token>")
```
Here is some logs where I have received `key verification event`, but there are no docs/examples how to handle it.
```log
INFO:niobot.client:Encryption support enabled automatically.
INFO:niobot.client:Auto-joining rooms enabled.
INFO:niobot.client:Auto-updating read receipts enabled.
INFO:niobot.client:Logging in with existing access token.
INFO:niobot.client:Performing first sync...
...
INFO:niobot.client:Starting sync loop
Logged in!
---> WARNING:nio.crypto.log:Received key verification event with an unknown transaction id from @<me>:matrix.org <---
...
```

All my messages that I send from bot are like these.
![image](https://github.com/user-attachments/assets/fac98654-e0c8-4526-b6fb-33e174dbfa6d)

![image](https://github.com/user-attachments/assets/c2f9a843-035e-4c67-8c8a-de6d53c399bc)

**Comment by nexy7574** [comment]
matrix-nio (and consequently nio-bot) does not support cross-signing (see: https://matrix-nio.readthedocs.io/en/latest/)

If you can't verify the session manually via text, take a look at [this](https://matrix-nio.readthedocs.io/en/latest/examples.html#manual-encryption-key-verification)

Generally, you cannot verify bot sessions, because you can't *actually* verify the identity of an automated program.

**Comment by zatevakhin** [comment]
@nexy7574 Thanks for help! Think this issue can be closed.
I have tried to use those examples to add handle key events but noticed that it is [already implemented](https://github.com/matrix-nio/matrix-nio/blob/3006c2253fcd86d69e7d0be146ab84e0f71f6c83/nio/client/async_client.py#L1410) in `matrix-nio`.
I don't know what issues I have with **Element Desktop**, but I can't verify the bot from it. It works for me from the mobile client. So, I just verified the bot with the mobile client, and it worked for me right away.
